packages/toast/README.md

```diff
@@ -0,0 +1,61 @@
+## Overview
+
+`sp-toast`s display brief, temporary notifications. They are noticeable but do not disrupt the user experience and do not require an action to be taken.
```

Comment:
could we add an example of disappearing as well

Reply:
@adixon-adobe @davidyxu Can the two of you please take a look at the [a11y doc](https://github.com/adobe/react-spectrum-v3/blob/master/packages/%40react-types/toast/src/index.d.ts#L9) and [this comment](https://github.com/adobe/spectrum-web-components/pull/306/files#r350175966) to help clarify how we'd like the API to come together before I add this example. Thanks!